exercises/leap/leap-test.rkt

```diff
@@ -10,9 +10,10 @@
      "leap tests"
 
      (test-eqv? "vanilla-leap-year" (leap-year? 1996) #t)
-     (test-eqv? "any-old-year" (leap-year? 1997)#f)
+     (test-eqv? "any-old-year" (leap-year? 2015)#f)
```

Comment:
Any particular reason the test year is changed?

Reply:
I basically put it in as it aligned the test data with https://github.com/exercism/problem-specifications/blob/master/exercises/leap/canonical-data.json.

I'm not wedded to the change so happy to revert...